UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/FunctionalTests/CallEssEndPoint.cs

```diff
@@ -17,13 +17,14 @@ public class CallEssEndPoint
         private string FssJwtToken { get; set; }
         private GetProductIdentifiers getproductIdentifier { get; set; }
 
-        private static readonly EssAuthorizationConfiguration ESSAuth = new TestConfiguration().EssAuthorizationConfig;
-        private static readonly FunctionalTestFSSApiConfiguration FSSAuth = new TestConfiguration().FssConfig;
+        private static readonly ESSApiConfiguration ESSAuth = new TestConfiguration().EssAuthorizationConfig;
+        private static readonly FSSApiConfiguration FSSAuth = new TestConfiguration().FssConfig;
```

Comment:
Is this ESS/FSS API Call Configuration or Authentication Configuration ?

Reply:
Updated.